fix.go

```diff
@@ -0,0 +1,135 @@
+package kubeaudit
+
+import (
+	"bytes"
+
+	"github.com/Shopify/kubeaudit/internal/k8s"
+	"github.com/Shopify/kubeaudit/internal/yaml"
+	"github.com/Shopify/kubeaudit/k8stypes"
+)
+
+func fix(results []Result) ([]byte, error) {
+	var outputBytes [][]byte
+	var newResources []k8stypes.Resource
+
+	if len(results) == 0 {
+		return []byte{}, nil
+	}
+
+	// Fix all the resources
+	for _, result := range results {
+		for _, auditResult := range result.GetAuditResults() {
+			newResources = append(newResources, auditResult.Fix(result.GetResource().Object())...)
+		}
+	}
+
+	// Convert all the resources to bytes
+	for _, result := range results {
+		if result.GetResource().Object() == nil {
+			outputBytes = append(outputBytes, result.GetResource().Bytes())
+			continue
+		}
+
+		fixedresourceBytes, err := resourceToBytes(result.GetResource().Object(), result.GetResource().Bytes())
+		if err != nil {
+			return nil, err
+		}
+
+		outputBytes = append(outputBytes, fixedresourceBytes)
+	}
+
+	// Convert all the new resources to bytes
+	for _, newResource := range newResources {
+		fixedresourceBytes, err := resourceToBytes(newResource, nil)
+		if err != nil {
+			return nil, err
+		}
+		outputBytes = append(outputBytes, fixedresourceBytes)
+	}
+
+	fixedManifest := bytes.Join(outputBytes, []byte("---"))
+
+	return fixedManifest, nil
+}
+
+func resourceToBytes(fixedResource k8stypes.Resource, origResourceBytes []byte) ([]byte, error) {
+	fixedresourceBytes, err := k8s.EncodeResource(fixedResource)
```

Comment:
`fixedResourceBytes`?

Reply:
It's the fixed resource, but in byte form. What do you think it should be called?